extras/rya.geoindexing/src/main/java/org/apache/rya/indexing/mongodb/geo/GeoMongoDBStorageStrategy.java

```diff
@@ -90,15 +113,18 @@ public GeoMongoDBStorageStrategy(final double maxDistance) {
 
     @Override
     public void createIndices(final DBCollection coll){
-        coll.createIndex("{" + GEO + " : \"2dsphere\"" );
```

Comment:
presumably yes, but it didn't really work either.  the BasicDBObject worked though